packages/local-cli/core/findPlugins.js

```diff
@@ -91,13 +89,11 @@ const findPluginsInFolder = folder => {
 
   const deps = union(
     Object.keys(pjson.dependencies || {}),
-    Object.keys(pjson.devDependencies || {}),
+    Object.keys(pjson.devDependencies || {})
   );
 
   return deps.reduce((acc, pkg) => {
-    let commands = acc.commands;
-    let platforms = acc.platforms;
-    let haste = acc.haste;
+    let { commands, platforms } = acc;
```

Comment:
`haste` could be retrieved here

Reply:
Yes, but this would cause another error: using "let" where "haste" is not re-assigned. Wanted to avoid double-destructing statement for the sake of it.